appserver/monitoring-console/webapp/src/main/webapp/js/mc-view-units.js

```diff
@@ -192,6 +208,20 @@ MonitoringConsole.View.Units = (function() {
       return number.toString().padStart(2, '0');
    }
 
+   const DECIMAL_NUMBER_PATTERN = '([0-9]+\.)?[0-9]+';
```

Comment:
Wouldn't be better to declate constants on one place like we do in Java? It's only a question, not request.

Reply:
The JS code uses the "directly called function module" pattern to do effective scoping. Constants are located in the "module" that uses them. Same as in java where we hide most constants in private fields of a class.